packages/front-end/src/app/app.tsx

```diff
@@ -1,13 +1,22 @@
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-import styles from './app.module.scss';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import AddPerson from './add-person-page';
+import FamilyTreeDisplay from './family-tree-page';
+import Login from './login-page';
+import Register from './register-page';
 
 export function App(): JSX.Element {
   return (
-    <div>
-      <span> Hello there, </span>
-      Welcome front-end 👋
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path='/'>
+          <Route index element={<FamilyTreeDisplay />} />
+          <Route path='/add-person' element={<AddPerson />} />
+          <Route path='/login' element={<Login />} />
+          <Route path='/register' element={<Register />} />
+        </Route>
+      </Routes>
+    </BrowserRouter>
```

Comment:
Just wondering about front-end BDD's cucumber integration, I think the router element needs to go outside of App (and into index) for it to work properly. Altough considering the attention we've given to front-end testing, not a major issue